src/utils/workflow-utils.ts

```diff
@@ -253,13 +261,9 @@ export const generateTaskSpec = (
                         },
                     };
 
-                    if (
-                        entityType === 'materialization' &&
-                        incompatibleSchemaChanges &&
-                        hasLength(incompatibleSchemaChanges[bindingUUID])
-                    ) {
+                    if (onIncompatibleSchemaChangeOverridden) {
```

Comment:
What is driving us doing these updates manually here and right up above instead of using `addOrRemoveOnIncompatibleSchemaChange`?